src/checker.rs

```diff
@@ -117,6 +154,11 @@ impl Checker for JavaCode {
     mk_checker!(is_func, MethodDeclaration);
     mk_checker!(is_func_space, Program, ClassDeclaration);
     mk_checker!(is_non_arg,);
+
+    #[inline(always)]
+    fn is_else_if(_: &Node) -> bool {
+        false
+    }
```

Comment:
Let's use https://doc.rust-lang.org/std/macro.unimplemented.html in the unimplemented `is_else_if`s.